src/pages/settings/Payments/TransferBalancePage.js

```diff
@@ -166,70 +167,84 @@ class TransferBalancePage extends React.Component {
         const isButtonDisabled = !isTransferable || !selectedAccount;
         const errorMessage = !_.isEmpty(this.props.walletTransfer.errors) ? _.chain(this.props.walletTransfer.errors).values().first().value() : '';
 
+        const shouldShowTransferView =
+            !_.isEmpty(this.props.userWallet) &&
```

Comment:
This check is not needed

Reply:
@Pujan92 We can remove `!_.isEmpty(this.props.userWallet)` and use `lodashGet` to get the `tierName`.

Reply:
done

Reply:
But `userWallet` is set to `{}` in the default props! So `this.props.userWallet` is always defined, and it won't throw "can't read property of undefined".